PalindromeLinkList

```diff
@@ -0,0 +1,75 @@
+//
+//  main.cpp
+//  Palindromo_linkedlist
+//
+//  Created by Cristina Jimenez on 8/25/17.
+//  Copyright © 2017 Cristina Jimenez. All rights reserved.
+//
+
+#include <iostream>
```

Comment:
This is a good start - feel free to finish this when you have time and ping me. I'll happily look over it.